### ISSUE TITLE
TODO

### ISSUE THREAD

**Issue by AppleDevelops** [ArufaTeam/Arufa#5]
- [ ] Complete API 3.0.0-ALPHA4
- [ ] Add travis
- [ ] Add circleci
- [ ] New logger
- [ ] Remove deprecated functions
- [ ] Bug fixes
- [ ] Features
- [ ] Code cleanup
- [ ] API 3.0.0-ALPHA5

Add more

**Comment by RateekMCPE** [comment]
👍 todo !

**Comment by Derpific** [comment]
I think we should remove deprecated functions first

**Comment by NFGamerMC** [comment]
@AppleDevelops You remember my old roll, correct? Well I can be the service manager as I was before. I can do Websites, Jenkins, Travis, Circle CI, Forums, Issue Managment (optional). I was a part then you all got mad at me because I was getting upset with LinuxLogo

**Comment by akaSnapy** [comment]
you may add enchanting table here :)

**Comment by DerddyDert** [comment]
Looking great so far!

**Comment by NFGamerMC** [comment]
@AppleDevelops @RateekMCPE @Derpific I have partially moved this into Projects to make it cleaner. https://github.com/ArufaTeam/Arufa/projects

**Comment by Fabiansigala** [comment]
@AppleDevelops can I help out I will help out with most of this Complete API 3.0.0-ALPHA4
 Add travis
 Add circleci
 New logger
 Remove deprecated functions
 Bug fixes
 Features
 Code cleanup
 API 3.0.0-ALPHA5
 New labels
 Remove Achievement system.
 CRITICAL > All tiles crash the server
 Fix sounds
 Resource packs

**Comment by Fabiansigala** [comment]
@AppleDevelops I have been accepted to join by @NFGamerMC I just need you to accept me.

**Comment by Fabiansigala** [comment]
@AppleDevelops I am wondering if I can join Arufa and help out I 

I am wondering if I am accepted?

**Comment by Fabiansigala** [comment]
@AppleDevelops I am willing to help out with 

 Complete API 3.0.0-ALPHA4
 Add travis
 Add circleci
 New logger
 Remove deprecated functions
 Bug fixes
 Features
 Code cleanup
 API 3.0.0-ALPHA5
 New labels
 Remove Achievement system.
 CRITICAL > All tiles crash the server
 Fix sounds
 Resource packs

**Comment by Pizzahaet** [comment]
Why remove the Achievement system? It can be toggled in the Server.Properties... I think you should keep it.

**Comment by daniktheboss** [comment]
True, but if you think about it it's useless. It should be implemented in vanilla mcpe client.
